packages/core/src/test-utils/use-fake-time.ts

```diff
@@ -19,7 +19,11 @@ export const advanceFakeTime = (milliseconds: number) => {
 export const testUsingFakeTime = (dateTime = "2015-10-21T07:28:00Z") => {
   usingFakeTime = true;
 
-  jest.useFakeTimers();
+  jest.useFakeTimers({
+    doNotFake: [
+      "nextTick",
+    ],
+  });
```

Comment:
Note: this is required to use `@hapi/shot`. I don't think this is that important to fake given that is it more about the event loop than timing anyway.

Reply:
Dictating how fake timers work globally because of an arbitrary library "hapi" is not ok. Either hapi is considered as having side-effects (and thus hidden behind an abstraction), or the few unit-tests that test hapi-framework can configure fake timers ad-hoc.